.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
what's dis file?

Reply:
oh! this is vscode's config for attaching the chrome debugger. it doesnt need to be committed. but it can be.

Reply:
aight, i don't mind either way :) just making sure it wasn't tagging along unbidden